uber/site_sections/jobs.py

```diff
@@ -172,6 +177,15 @@ def staffers_by_job(self, session, id, message=''):
                                 .order_by(Attendee.full_name).all()
         }
 
+    def allStaffers(self, session, id, message=''):
+        job = session.job(id);
```

Comment:
That function actually isn't even supposed to exist... was a dummy function for troubleshooting, just removing the whole thing good catch!